src/__tests__/Carousel.tsx

```diff
@@ -43,8 +43,8 @@ describe('Slider', function() {
 
         componentInstance = component.instance();
 
-        totalChildren = children && children.length ? React.Children.count(componentInstance.props.children) : 0;
-        lastItemIndex = totalChildren - 1;
+        totalChildren = children ? React.Children.count(componentInstance.props.children) : 0;
```

Comment:
`React.Children.count` accepts empty array and would return 0 in that case anyway.